### PR TITLE
DLP Client (GAPIC-only)

### PR DESCRIPTION
**Pull request by lukesneeringer**
@ace-n This should be what you need for samples.

### CODE REVIEW

File: dlp/docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
```
Comment (jonparrott) [inline dlp/docs/conf.py]: We should separately discuss these generated docs configs, unless you want me to get into it in this PR.

File: dlp/docs/starting.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Using the API
+-------------
+
+
+Authentication
+~~~~~~~~~~~~~~
+
+To authenticate all your API calls, first install and setup the `Google Cloud SDK`_.
```
Comment (jonparrott) [inline dlp/docs/starting.rst]: We gotta get rid of this and just link to https://cloud.google.com/auth

File: dlp/google/cloud/dlp.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+from google.cloud.dlp_v2beta1 import enums
+from google.cloud.dlp_v2beta1 import types
+from google.cloud.dlp_v2beta1 import DlpServiceClient
```
Comment (jonparrott) [inline dlp/google/cloud/dlp.py]: Was the vision one `VisionServiceClient` or just `VisionClient`?

Comment (lukesneeringer) [inline dlp/google/cloud/dlp.py]: The Vision one is spelled `ImageAnnotatorClient`. It seems to use the name in the proto with `Client` appended.

File: dlp/docs/starting.rst (hunk 2 above)
Comment (lukesneeringer) [inline dlp/docs/starting.rst]: What is the actual link? That one 404s.

Comment (lukesneeringer) [inline dlp/docs/starting.rst]: (I found this out when I went to go make that change. ;-))

**Pull request closed without merging**